R/simulation-results.R

```diff
@@ -52,7 +52,7 @@ SimulationResults <- R6::R6Class(
       individualIds <- c(individualIds)
       values <- rClr::clrCall(self$ref, "AllValuesFor", path, as.integer(individualIds))
 
-      if (unique(is.nan(values)) && stopIfNotFound) {

```

Comment:
This would only evaluate the first entry in the array. In our case, it would return FALSE, TRUE therefor the error
In fact, we should use all here

Reply:
FYI @Yuri05